src/main/java/Ui.java

```diff
@@ -0,0 +1,28 @@
+public class Ui {
+    public static void printLine() {
+        System.out.println("____________________________________________________________");
+    }
+
+    public static void greetUser() {
+        String logo = " ____        _        \n"
+                + "|  _ \\ _   _| | _____ \n"
+                + "| | | | | | | |/ / _ \\\n"
+                + "| |_| | |_| |   <  __/\n"
+                + "|____/ \\__,_|_|\\_\\___|\n";
+
+        System.out.println("Hello from\n" + logo);
+        printLine();
+        System.out.println("Hello! I'm Duke");
+        System.out.println("What can I do for you?");
+        printLine();
+    }
+
+    public static void sayBye() {
+        printLine();
+        System.out.println("Bye. Hope to see you again soon! :)");
+        printLine();
+    }
+
+    public static void printErrorMessage() {
```

Comment:
Is this an incomplete method? Maybe remove it if it is not being used.

Reply:
Thank you for your feedback! @sevenseasofbri I'll work on them :)